src/explore-education-statistics-admin/src/components/editable/EditableContentForm.tsx

```diff
@@ -76,20 +78,29 @@ const EditableContentForm = ({
     onIdle,
   });
 
-  const validateElements = useCallback((elements: Element[]) => {
-    let error: string | undefined;
-
-    elements.some(element => {
-      if (element.name === 'image' && !element.getAttribute('alt')) {
-        error = 'All images must have alternative (alt) text';
-        return true;
-      }
-
-      return false;
-    });
-
-    return error;
-  }, []);
+  function isInvalidImage(element: Node) {
+    return (
+      (element.name === 'imageBlock' || element.name === 'imageInline') &&
+      !element.getAttribute('alt')
+    );
+  }
+
+  const validateElements = useCallback(
```

Comment:
Whilst it's valid to pop this error during `onBlur` validation, I wonder if there's anything we can do to re-validate when the image changes i.e. alt text is added?

Users could potentially find it confusing that adding alt text doesn't immediately make the error go away.

Reply:
The form was set to not revalidate on change, I couldn't see any reason why this is necessary and have checked the UI tests, so have removed it. Now it'll remove this and the 'Enter content' errors when they no longer apply.